scripts/unicode.py

```diff
@@ -106,6 +122,39 @@ def _load_unicode_data(self):
             if category == 'M' or 'M' in expanded_categories.get(category, []):
                 self.general_category_mark.append(char_int)
 
+    def _load_standardized_variants(self):
+        for line in self._fetch("StandardizedVariants.txt").splitlines():
+            strip_comments = line.split('#', 1)[0].strip()
+            if not strip_comments:
+                continue
+
+            pieces = strip_comments.split(';')
```

Comment:
you could do something like
```
variation_sequence, description, differences = strip_comments.split(';')
description = description.strip()
```
and that will throw an exception if it doesn't have three pieces